pkg/engine/crds.go

```diff
@@ -41,6 +41,10 @@ func Ensure(config *rest.Config) error {
 	if err != nil && !errors.IsAlreadyExists(err) {
 		return fmt.Errorf("error creating the Endpoint CRD: %s", err)
 	}
+	_, err = apiext.ApiextensionsV1beta1().CustomResourceDefinitions().Create(getMcsCRD())
```

Comment:
please use utils.CreateOrUpdateCrd, see
https://github.com/submariner-io/submariner-operator/commit/753b7003b4b02905815fd68da660dc0e1908067c#diff-07a74df9a382711c002273a217a37de4R50

Reply:
Done , this has been now moved to pkg/lighthouse/crds.go, made the changes there.